infra/terraform/modules/atlantis_iam_role/main.tf

```diff
@@ -0,0 +1,84 @@
+data "aws_caller_identity" "current" {}
+
+# Some awful nonsense to get the Atlantis ECS role name from its ARN.
+# The Atlantis module outputs the role ARN, but Terraform resources
+# take the role name as an argument, not ARN
+locals {
+  ecs_role_arn_parts = "${split("/", var.atlantis_ecs_role_arn)}"
+  ecs_role_name      = "${element(local.ecs_role_arn_parts, length(local.ecs_role_arn_parts) - 1)}"
+}
+
+data "aws_iam_policy_document" "atlantis_terraform" {
+  statement {
+    actions   = ["ec2:*"]
```

Comment:
If I understand correctly for now Atlantis will only be able to perform EC2 actions (e.g. create/destroy "servers" and related stuff) right? (a part from being able do manage TF state S3 bucket)

I guess if we want Atlantis to apply any possible TF changes it will need pretty much root access as you never know what we'll put in Terraform (e.g. DBs, Lambdas, Athena, Glue stuff and god knows what else), is that the case?